internal-packages/ai/src/tools/check-generated-schemas.ts

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env tsx
+
+/**
+ * Runtime check to ensure generated schemas are up-to-date
+ * This runs during app startup to catch stale schemas early
+ */
+
+import { existsSync, readFileSync } from 'fs';
+import path from 'path';
+import { createHash } from 'crypto';
+
+// Import all tools to check their current state
+import checkSpellingGrammarTool from './check-spelling-grammar';
+import extractFactualClaimsTool from './extract-factual-claims';
+import factCheckerTool from './fact-checker';
+import checkMathWithMathJsTool from './check-math-with-mathjs';
+import checkMathTool from './check-math';
+import checkMathHybridTool from './check-math-hybrid';
+import extractMathExpressionsTool from './extract-math-expressions';
+import extractForecastingClaimsTool from './extract-forecasting-claims';
+import documentChunkerTool from './document-chunker';
+import fuzzyTextLocatorTool from './fuzzy-text-locator';
+import { detectLanguageConventionTool } from './detect-language-convention';
+import forecasterTool from './forecaster';
+import { linkValidator } from './link-validator';
+import perplexityResearchTool from './perplexity-research';
```

Comment:
Do we really need to in import each one of these individually? Maybe there's a way we could do a group import. This is because I assume we're going to want this in a few different places.